ouroboros-consensus/docs/VersioningSchemeDecision.md

```diff
@@ -0,0 +1,54 @@
+# Choosing a Versioning Schemes
+
+This document records our discussions around choosing how to version our packages.
+
+In its current state, the document is primarily focused on one question: for each individual package, how should the version listed in the .cabal file on the `master` branch relate to the version in the .cabal file in the latest release?
+In particular, I think this question is almost entirely independent of usings `git` branches versus `git` tags, SemVer versus PVP, etc.
+This document should eventually specify that level of detail as well.
+
+## Desiderata
+
+- *Conventional Version Number*.
+  We'd like our release versions to have the standard shape of `A.B.C`, where in particular increments of `C` principally represent bugfixes.
```

Comment:
I think this does bear on PVP, since PVP has two major versions so `C` would be a minor version rather than a patch version.

Reply:
For the record, I think 2 dimensions versus 3 dimensions is a superficial difference. The differences discussed at https://pvp.haskell.org/faq/#how-does-the-pvp-relate-to-semantic-versioning-semver all seem secondary to me.

Reply:
I've spelled addressed this somewhat-directly in the Desiderata section now.